src/pages/open-web-fellows/fellows2017.js

```diff
@@ -0,0 +1,197 @@
+            </FellowBlock>
+          </ContentContainer>
+          <ContentContainer className="grey slant">
+            <FellowBlock
+              name="Freddy Martinez"
+              image="/assets/fellows2017/Freddy@2x.jpg"
+              company="Freedom of the Press Foundation"
+              companyHref="https://freedom.press/"
+              handle="B_Meson"
+              handleHref="https://twitter.com/B_Meson"
+            >
+              Freddy is a activist, security engineer, and was previously a physicist. He is also the Director of Lucy Parsons Labs, a police accountability and transparency non-profit based in Chicago. His interests include computer security, hacking, politics and open source software.
+            </FellowBlock>
+          </ContentContainer>
+          <ContentContainer>
+            <FellowBlock
+              name="Rishab Nithyanand"
+              image="/assets/fellows2017/Rishab@2x.jpg"
+              company="Data & Society"
+              companyHref="http://www.datasociety.net/"
+              handle="rishabn_"
+              handleHref="https://twitter.com/rishabn_"
+              linkName="Rishab’s website"
+              link="www.rishabn.com"
+            >
+              Rishab has research interests in the areas of security, privacy, and Internet measurement. More specifically, he enjoys research geared towards defending anonymity, understanding Internet censorship, measuring aspects of the advertising and tracking ecosystem, and generally understanding the impact of the Internet on the sociopolitical realities of today. In the past, he dabbled in usable security and computer theory. In the coming year, he plans to work towards understanding of how the Internet impacts political action and discourse.
+            </FellowBlock>
+          </ContentContainer>
+          <ContentContainer className="grey slant">
+            <FellowBlock
+              name="Rebecca Ricks"
+              image="/assets/fellows2017/Rebecca@2x.jpg"
+              company="Human Rights Watch"
+              companyHref="https://www.hrw.org/"
+              handle="baricks"
+              handleHref="https://twitter.com/baricks"
+              link="http://beccaricks.com"
+              linkName="Rebecca's website"
+            >
+              Rebecca is a multidisciplinary software developer, researcher, and advocate who is engaged in issues of privacy and freedom of expression online. Drawing on a background in Middle East politics and Arabic, she has conducted research at New York University on algorithmic and surveillance systems, with a particular emphasis on government repression. A self-taught programmer, she is passionate about building digital tools for activists that promote free speech on the internet.
+            </FellowBlock>
+          </ContentContainer>
+          <ContentContainer className="grey slant">
```

Comment:
`ContentContainer className="grey slant"` these classnames should probably be staggered to remove the weirdness in the last 5 fellow bios.